puppet/instrumental_tools/templates/instrumental.yml.erb

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Oh, I replied before the shrug thing.

In general, I like config files more than command lines and environments, but the things here are all single value default sort of options that are probably just fine at the command line.  Once we need to specify more complex arguments, it will be time to think about config file only options.  The defaults and simple arguments being accessible from the command line forever is probably a good thing.

TLDR: needs more discussion outside of this PR, ship it.